node.js

```diff
@@ -258,5 +265,24 @@ module.exports = {
   clearCaches: function clearCaches () {
     filenessCache = { }
     configCache = { }
+  },
+
+  testCanIUse: function testCanIUse (latestVersion) {
+    var now = Date.now()
+    var digitsDiff = String(now).length - String(latestVersion).length
```

Comment:
Why we need this variable?

Reply:
caniuse release_date has 10 digits while js Date in milliseconds has 13

Reply:
Because `release_date` contains seconds, and `Date.now` milliseconds. You algorithm too complex and will not work in some cases (like 999 in `release_date` and 1000_000 in `Date.now`)